docs/_layouts/default.html

```diff
@@ -89,6 +89,19 @@ <h2 class="text-delta">Table of contents</h2>
       </div>
     </div>
   </div>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.5.0/mermaid.min.js"></script>
```

Comment:
should I put this in our local assets folder?

Reply:
mhm, its just the docu, so I don't think so.

Reply:
but maybe it's better (data privacy wise)

Reply:
okay done 👍